server.js

```diff
@@ -56,6 +56,25 @@ app.get('/repos/:user', async (req, res) => {
 		.send(await request(`https://img.shields.io/badge/Repos-${response.public_repos}-brightgreen${req.originalUrl.slice(req.originalUrl.indexOf('?'))}`).raw())
 })
 
+app.get('/gists/:user', async (req, res) => {
+	const { user } = req.params;
+
+	const response = await request(
+		`https://api.github.com/users/${user}/gists`
+	).header(githubHeaders).json();
+
+	if (response.length === 0)
+		return res.status(404).send(response);
+
+	const badge = await request(
+		`https://img.shields.io/badge/Gists-${response.length}-brightgreen${req.originalUrl.slice(req.originalUrl.indexOf('?'))}`)
+		.raw();
+
+	res.contentType('image/svg+xml')
+		.header('Cache-Control', 'no-age', 'max-age=600')
+		.send(badge);
```

Comment:
No need to store the badge in a constant, you can just follow what's been done in the other endpoints.

```suggestion
		.send(await request(
		`https://img.shields.io/badge/Gists-${response.length}-brightgreen${req.originalUrl.slice(req.originalUrl.indexOf('?'))}`)
		.raw());
```